language/move-binary-format/src/unit_tests/deserializer_tests.rs

```diff
@@ -222,7 +222,7 @@ fn malformed_simple() {
 }
 
 // Ensure that we can deserialize a script from disk
-static EMPTY_SCRIPT: &[u8] = include_bytes!("../../../../types/src/test_helpers/empty_script.mv");
+static EMPTY_SCRIPT: &[u8] = include_bytes!("empty_script.mv");
```

Comment:
So this change duplicates the `empty_script.mv` binary file in the repository? That's a shame, since (1) we'll have to update both of these when the binary format changes, but we don't have clear instructions on how to do so, and (2) some conventional "wisdom" I've read on the Internet would have me believe that managing binary files with Git is a "Bad Idea." But, in this case the files are named such that it's easy to guess how to generate them, and the files are so small (18 bytes, smaller than most plain text files in this repository) that they shouldn't slow Git down.

Anyway, it's not a big deal. I guess the "rule of three's" superstition would only have us think about how to remove this duplication once we have _three_ spots in the repository that use an empty script bytecode file.

Reply:
Yeah this is a really good point. The justification I gave to myself was that `langauge` will soon become a separate repo, so duplications like this may become a bit more acceptable. Still, I fully agree with you that this is less than ideal and we should try to come up with a better way to hand this.

> Anyway, it's not a big deal. I guess the "rule of three's" superstition would only have us think about how to remove this duplication once we have three spots in the repository that use an empty script bytecode file.

I wonder if we should just get rid of these binary files and generate the empty script on the fly. Although I vaguely remember brining this up a long time ago, but ran into some concerns/issues at the time.

Reply:
> I wonder if we should just get rid of these binary files and generate the empty script on the fly.

Yes, I also thought about that possibility... one downside I see is that the test ends up testing the binary file generation. But that seems like a pretty minor issue. I wonder what else was brought up?

In any case, I don't think it's crucial to deal with this now :)